backend/config/sync/user-role.authenticated.json

```diff
@@ -18,6 +24,12 @@
     {
       "action": "api::event.event.findOne"
     },
+    {
```

Comment:
I wonder why there are two of the same announcement apis could you delete one?

Reply:
I removed one of them